torchvision/transforms/transforms.py

```diff
@@ -1130,11 +1128,13 @@ def forward(self, tensor: Tensor) -> Tensor:
         tensor = transformed_tensor.view(shape)
         return tensor
 
-    def __repr__(self):
-        format_string = self.__class__.__name__ + "(transformation_matrix="
-        format_string += str(self.transformation_matrix.tolist()) + ")"
-        format_string += ", (mean_vector=" + str(self.mean_vector.tolist()) + ")"
-        return format_string
+    def __repr__(self) -> str:
```

Comment:
This is not equivalent, but it seems to me that the original implementation was wrong. I will ask the reviewer to double check this, as my interpretation could be wrong.

For example if the original output would is

`class_name(transformation_matrix=[1, 2, 3, 4]), (mean_vector=[2, 3, 4, 5])`

with the modifications we should expect


`class_name(transformation_matrix=[1, 2, 3, 4], mean_vector=[2, 3, 4, 5])`

Reply:
Agreed, the old is wrong.